### PR TITLE
Users implement crud

### PR DESCRIPTION
**Pull request by kabust**

### CODE REVIEW

File: user/models.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        if extra_fields.get("is_staff") is not True:
+            raise ValueError("Superuser must have is_staff=True.")
+        if extra_fields.get("is_superuser") is not True:
+            raise ValueError("Superuser must have is_superuser=True.")
+
+        return self._create_user(email, password, **extra_fields)
+
+
+class User(AbstractUser):
```
Comment (aLEKS-e3) [inline user/models.py:43]: add `__str__` with email

File: user/serializers.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
```
Comment (aLEKS-e3) [inline user/serializers.py:6]: add a blank line between classes

File: user/tests/test_user.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
+        self.assertIn("access", res.data)
+
+    def test_verify_token(self):
+        payload = {"token": self.access}
+        url = reverse("user:token_verify")
+        res = self.client.post(path=url, data=payload)
+
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
```
Comment (aLEKS-e3) [inline user/tests/test_user.py:66]: I think it's no need to test token endpoints since they're built-in, but you could test a password validation with 5 symbols minimum

**Review by aLEKS-e3 — CHANGES_REQUESTED** [review]

**Review by aLEKS-e3 — APPROVED** [review]
good job!

**Review by AntonSkyrda — APPROVED** [review]
Good Job !
